src/index.js

```diff
@@ -40,6 +41,9 @@ const init = () => {
       event.currentTarget.value = '';
     }
   });
+  clearAllBtn.addEventListener('click', () => {
+    ToDoListArray.clearAllCompleted(ulList);
+  });
```

Comment:
- [ ] Please make sure that your 'Clear All Completed' button is working since this function is not currently working.

Reply:
This function work properly but first you need to check a task